internal/jitsi/jibri.go

```diff
@@ -143,43 +201,43 @@ func (j *Jibri) setPV(sts *appsv1.StatefulSetSpec) {
 }
 
 func (j *Jibri) preparePVC() v1.PersistentVolumeClaim {
-	if j.Storage.PVC.Kind == "" {
-		j.Storage.PVC.Kind = "PersistentVolumeClaim"
+	if j.Spec.Storage.PVC.Kind == "" {
+		j.Spec.Storage.PVC.Kind = "PersistentVolumeClaim"
 	}
-	if j.Storage.PVC.APIVersion == "" {
-		j.Storage.PVC.APIVersion = "v1"
+	if j.Spec.Storage.PVC.APIVersion == "" {
+		j.Spec.Storage.PVC.APIVersion = "v1"
 	}
-	if j.Storage.PVC.Name == "" {
-		j.Storage.PVC.Name = volumeName(j.name)
+	if j.Spec.Storage.PVC.Name == "" {
+		j.Spec.Storage.PVC.Name = volumeName(j.name)
 	}
-	if j.Storage.PVC.Spec.AccessModes == nil {
-		j.Storage.PVC.Spec.AccessModes = []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce}
+	if j.Spec.Storage.PVC.Spec.AccessModes == nil {
+		j.Spec.Storage.PVC.Spec.AccessModes = []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce}
 	}
 	return v1.PersistentVolumeClaim{
 		TypeMeta: metav1.TypeMeta{
-			Kind:       j.Storage.PVC.Kind,
-			APIVersion: j.Storage.PVC.APIVersion,
+			Kind:       j.Spec.Storage.PVC.Kind,
+			APIVersion: j.Spec.Storage.PVC.APIVersion,
```

Comment:
Default values for  TypeMeta should be already correct.

Reply:
removed